test/e2e/e2e_test.go

```diff
@@ -25,6 +25,9 @@ const (
 	// simPdDeployment references the YAML file for the deployment
 	// running the vLLM simulator with PD
 	simPdDeployment = "./yaml/vllm-sim-pd.yaml"
+	// simDpDeployment references  the YAML file for the deployment
```

Comment:
nit: idiomatic Go should capitalize the acronym as `DP` not `Dp` (same as `HTTP`, `TCP`, etc.)